packages/legend-query-builder/src/stores/__tests__/TEST_DATA__QueryBuilder_PreviewDataTest.ts

```diff
@@ -738,35 +738,35 @@ export const TEST_DATA__PreviewData_lambda_numeric = {
               values: [
                 {
                   _type: 'string',
-                  value: 'Count',
+                  value: 'Count (count)',
```

Comment:
why are we changing this. is the default query we show for preview values changing??

Reply:
thanks! I corrected the code and remove these changes